install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -13,567 +14,208 @@ toc::[]
 
-.Architecture - Red Hat Gluster Storage Container Converged with OpenShift
-image::OpenShift_Containerization_Gluster_412816_0716_JCS_converged.png["Architecture - Red Hat Gluster Storage Container Converged with OpenShift"]
+[[overview-external-glusterfs]]
+=== {gluster-external}
 
-Step-by-step instructions for this containerized solution are provided
-separately in the following Red Hat Gluster Storage documentation:
+include::install_config/persistent_storage/topics/glusterfs_overview_external.adoc[]
 
-link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/[Container-Native Storage for OpenShift Container Platform]
+[[overview-standalone-glusterfs]]
+=== Standalone {gluster}
 
-[[container-native-storage-recommendations]]
-=== Container Native Storage Recommendations
+include::install_config/persistent_storage/topics/glusterfs_overview_standalone.adoc[]
 
-{product-title} offers container native storage (CNS) storage, which makes it
-easier for {product-title} users to fulfill their storage needs. With CNS,
-solution users and administrators are empowered to have storage and application
-pods running together on the same infrastructure and sharing the same resources.
+[[overview-volumes]]
+=== GlusterFS Volumes
 
-See link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/[Container-Native Storage for OpenShift Container Platform] for configuring CNS as part of an
-{product-title} cluster.
+include::install_config/persistent_storage/topics/glusterfs_overview_volumes.adoc[]
 
-[[volumes-creation-time-for-container-native-storage]]
-==== Creation Time of Volumes with Container Native Storage
+[[overview-block-volumes]]
+=== gluster-block Volumes
 
-Building environment storage can influence the time it takes for an application
-to start. For example, if the application pod requires a persistent volume claim
-(PVC), then extra time might have to be considered for CNS to be created and
-bound to the corresponding PVC. This effects the build time for an application
-pod to start.
+include::install_config/persistent_storage/topics/glusterfs_overview_block_volumes.adoc[]
 
-Creation time of CNS volumes scales linearly up to 100 volumes. In the latest
-tests, each volume took approximately 6 seconds to be created, allocated, and
-bound to a pod.
+[[overview-s3-storage]]
+=== Gluster S3 Storage
 
-All tests were performed on one trusted storage pool (TSP), using hardware
-configuration for CNS per the link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/[Container-Native Storage for OpenShift Container Platform] guidelines.
+include::install_config/persistent_storage/topics/glusterfs_overview_s3_storage.adoc[]
 
```

Comment:
The overview sections should not contain any examples. That belongs further down the page. That said, I could see adding more content under "Provisioning" for gluster-block and gluster-s3.

However, currently we don't support gluster-block for anything other than Logging and Metrics, and that is done via the advanced installer, so I would think documenting on how to use it generally will lead to some angry customers wondering why they're not supported. 

Additionally, gluster-s3 is currently only in tech preview. The advanced installer can deploy a single instance of a gluster-s3 service but this may not be how we actually want it to be deployed. I believe the currently preferred option would be to use the Service Catalog to deploy it. At that point, using the gluster-s3 storage falls to user applications using S3 credentials they define or are provided to them.

Reply:
We can't/should not say we support a function only for certain products. 

The function should be, as the documents state, be to support a usecase. 

If the TP state is not a true reprentation of what we want to deploy, I wonder if we should be considering this for TP status.

Reply:
Why can't we say that? The only workloads that we've verified on gluster-block are those of ElasticSearch and Cassandra. Is there a different way you want to say that?